src/lib/users/clients/users_finish.rb

```diff
@@ -93,6 +95,12 @@ def target_config
       @target_config
     end
 
+    def check_ids
+      validator = Y2Users::IdsValidator.new(Y2Users::ConfigManager.instance.target)
```

Comment:
I think we should use `target_config` (which is already merged with system config) instead of `Y2Users::ConfigManager.instance.target`. Otherwise, it will not detect ids duplicity with existing users/groups.

Reply:
I think it is job of merger to avoid such duplicitites. That is reason to use user config  and not result of merger. But if you think it will be better to report it to user in advance also about system collision, I can pass that.

Reply:
The merger merges users/groups by name, but it checks nothing about ids. I think it would be nice to notice to the user in advance if there is an id duplicity with the existing users/groups.

Reply:
Ok, I got the point, the autoyast merger reuses the id.